systers_portal/templates/common/sponsor_us.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <div class="container col-md-8 col-md-offset-2">
+    <h1> Sponsor Systers </h1>
```

Comment:
you would want to pick this as a variable, this would be per meetup, also the email id would be which they submitted in the form while creating a meetup and not Systers.

Reply:
There's only one Sponsor us page for all of the meetups in the meetup location. Should I change it to "Sponsor Meetup Location" ?

Reply:
@MeepyMay we would need your inputs here, possible options:
1. Have one sponsor us page on main header to request for sponsor for Systers? not sure if this is needed and other one per meetup.
2. Only have Meetup location -> Meetup -> Sponsor Us and help organizers w/ enabling sponsor ship request for them, no sponsor Systers page.

@sa7i ah, here if we go to a Meetup Location -> Meetup -> Sponsor Us, we would see show `Sponsor Systers` page?

Reply:
@MeepyMay we would need your inputs here, could you share which seems good to go with, possible options:
1. Have one sponsor us page on main header to request for sponsor for Systers? not sure if this is needed and other one per meetup.
2. Only have Meetup location -> Meetup -> Sponsor Us and help organizers w/ enabling sponsor ship request for them, no sponsor Systers page.

@sa7i per my understanding, here if we go to a Meetup Location -> Meetup -> Sponsor Us, we would see show Sponsor Systers page and in this case we should change it to Sponsor Meetup and/or location.

Reply:
@tapasweni-pathak Option #1. 

If they want to sponsor a specific community/meetup location such as Latinas or Pune, I'll include a notes option.

For "Sponsor Us," I will create an event with our new software (in AnitaB.org) CVENT and then we can embed a form here. 

For now, move forward with the lorem ipsum data and I'll create a new issue when the CVENT is created on my end. :)